### ISSUE TITLE
Hashing algo Argon2d

### ISSUE THREAD

**Issue by mkschulze** [GenieFramework/GenieAuthentication.jl#4]
Just posting here to rise attention to an interesting hashing algorythm.
https://www.argon2d.com/
Supposed to be superior than SHA-256, I heard.

**Comment by essenciary** [comment]
Nice! Is the a Julia library for it?

**Comment by mkschulze** [comment]
No, I don't think so. But maybe it could be just generated by using the C code and then https://github.com/JuliaInterop/Clang.jl ?

**Comment by mkschulze** [comment]
I gave it a go and this is the early result, not sure how to go forward tho: https://github.com/mkschulze/argon2.jl

**Comment by mkschulze** [comment]
I'm not able to work this out sadly.
